core/values/blockchain.go

```diff
@@ -0,0 +1,10 @@
+package values
+
+const (
+	// BeaconChainShardID is the ShardID of the BeaconChain
+	BeaconChainShardID = 0
+	// VotingPowerReduceBlockThreshold roughly corresponds to 3 hours
+	VotingPowerReduceBlockThreshold = 1350
+	// VotingPowerFullReduce roughly corresponds to 12 hours
+	VotingPowerFullReduce = 4 * VotingPowerReduceBlockThreshold
```

Comment:
no need for these threshold, we can just use the voting power == 0 to turn the validator to inactive. After four 1/4 voting power cut (or cumulatively 12 hours down time), the validator will be slashed.